pkg/controllers/node/termination/controller.go

```diff
@@ -77,13 +78,24 @@ func (c *Controller) Finalize(ctx context.Context, node *v1.Node) (reconcile.Res
 	if !controllerutil.ContainsFinalizer(node, v1beta1.TerminationFinalizer) {
 		return reconcile.Result{}, nil
 	}
+
+	nodeGracePeriodExpirationTime, err := c.terminationGracePeriodExpirationTime(ctx, node)
+	if err != nil {
+		return reconcile.Result{}, fmt.Errorf("couldn't evaulate node's terminationGracePeriod, %w", err)
```

Comment:
```suggestion
		return reconcile.Result{}, fmt.Errorf("evaluating node's terminationGracePeriod, %w", err)
```

nit: we're typically using gerunds for errors in our logs